### PR TITLE
Ammo num, autotagging and minor fixes.

### PR DESCRIPTION
**Pull request by skipor**

### CODE REVIEW

File: components/phttp/base.go [hunk 1]
```diff
@@ -46,37 +59,45 @@ func (b *Base) Shoot(ctx context.Context, ammo Ammo) {
...
 		if err != nil {
-			zap.L().Warn("Connect fail", zap.Error(err))
+			b.Log.Warn("Connect fail", zap.Error(err))
 			return
 		}
 	}
 
 	req, sample := ammo.Request()
+	if b.Config.AutoTag > 0 {
+		sample.AddTag(autotag(b.Config.AutoTag, req.URL))
```
Comment (direvius) [inline components/phttp/base.go]: Maybe we should only add auto tag if it was not specified in ammo file? Is it simple to do so?

File: components/phttp/ammo/simple/uri/decoder.go [hunk 2]
```diff
@@ -37,15 +37,15 @@ func newDecoder(ctx context.Context, sink chan<- *simple.Ammo, pool *sync.Pool)
 
 func (d *decoder) Decode(line []byte) error {
 	if len(line) == 0 {
-		return stackerr.Newf("empty line")
+		return errors.New("empty line")
```
Comment (direvius) [inline components/phttp/ammo/simple/uri/decoder.go]: We ignore empty lines in provider's [main cycle](https://github.com/skipor/pandora/blob/ee18af8390fd1e59d4bf98e7c1f5d3a4b0b02a52/components/phttp/ammo/simple/uri/provider.go#L52) (and this is OK). This code must never be executed. Maybe panic here?

Comment (skipor) [reply components/phttp/ammo/simple/uri/decoder.go]: The error will cancel shooting anyway but will contain more context. Line number for example.

File: components/phttp/base.go (hunk 1 above)
Comment (skipor) [reply components/phttp/base.go]: Put auto tag configuration under `auto-tag` scope, and add `no-tag-only` option with true by default.

**Comment by skipor** [comment]
Make Pandora CI build using `glide.lock` dependencies only. So PR that have not update `glide.lock` will have failed travis-ci check.

**Pull request merged**